src/app/content/components/Page/searchHighlightManager.ts

```diff
@@ -75,10 +74,9 @@ const selectResult = (services: Services, previous: HighlightProp | null, curren
     allImagesLoaded(services.container).then(
       () => {
         const target = selectedElements[0] as HTMLElement;
-        const container = target.closest('[tabindex]') as HTMLElement;
+        const focusTarget: HTMLElement | null = target.querySelector('[tabindex="0"]');
 
-        scrollTo(target);
```

Comment:
Is this `scrollTo()` just gone now? I noticed it was gone from the tests too.

Reply:
Yes. `focus` will scroll an element into view unless you [specify that it shouldn't](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus#preventscroll).